src/fft_2.h

```diff
@@ -64,9 +64,7 @@ Size2<T>::Size2(const gf::Field<T>& gf) : FourierTransform<T>(gf, 2)
 }
 
 template <typename T>
-Size2<T>::~Size2()
-{
-}
+Size2<T>::~Size2() = default;
```

Comment:
minor remark: it could be declared inside the class directly, same for the others